vpr/src/pack/cluster_router.cpp

```diff
@@ -545,13 +551,15 @@ bool try_intra_lb_route(t_lb_router_data* router_data,
 
 /* Creates an array [0..num_pb_graph_pins-1] lookup for intra-logic block routing.  Given pb_graph_pin id for clb, lookup atom net that uses that pin.
  * If pin is not used, stores OPEN at that pin location */
```

Comment:
This comment should be made Doxygen compatible and moved to the .h file

Reply:
It would be good to beef it up to include documentation of the parameters too if you understand them all.  If not, do what you can :).
Is the type t_pb_routes the specified array?  If so, good to mention it in the comment as right now the return type you document isn't an obvious match for the function signature.

Reply:
Added the following comment:

```
/**
 * @brief Creates an array [0..num_pb_graph_pins-1] for intra-logic block routing lookup. 
 * Given a pb_graph_pin ID for a CLB, this lookup returns t_pb_route corresponding to that
 * pin.
 *
 * @param intra_lb_nets Vector of intra-logic block nets.
 * @param logic_block_type Logic block type of the current cluster.
 * @param intra_lb_pb_pin_lookup Intra-logic block pin lookup to get t_pb_graph_pin from a pin ID.
 * @return t_pb_routes An array [0..num_pb_graph_pins-1] for intra-logic block routing lookup.
 */
```